shared/metrics/src/test/java/io/pravega/shared/metrics/StatsLoggerProxyTest.java

```diff
@@ -139,14 +139,18 @@ private TestMetric create(String name) {
         }
     }
 
+    //TODO: get rid of this class.
     private static class TestMetric implements OpStatsLogger, Counter, Meter, Gauge {
         @Getter
-        private final String name;
+        private final io.micrometer.core.instrument.Meter.Id id;
```

Comment:
Import Meter?

Reply:
This is inner static class, seems cannot import to the parent class.

Reply:
You should be able to import the parent class and refer to it as `Meter.Id` assuming there is not another class named `Meter` being imported.

Reply:
In Pravega Meter is a concrete metric type (Rate in more general sense) on top of Metric, which is the base class of all metric types; with Micrometer Meter is the the base class of all metrics type.
So there is indeed a class name collision.